src/test/java/org/typesense/api/DocumentsTest.java

```diff
@@ -60,6 +60,22 @@ void testCreateDocument() throws Exception {
         assertEquals("1", resp.get("id"));
     }
 
+    @Test
+    void testUnknownPresetsDoNotFailSearches() throws Exception {
```

Comment:
I'm not sure if keeping this test makes sense. But it's all I could think of for testing, because I don't think there's any way to setup a preset in these tests.